dm/dm-ansible/scripts/dm.json

```diff
@@ -1,40 +1,4 @@
 {
-  "__inputs": [
```

Comment:
what's the effects of these missing lines 🤔

Reply:
found [this](https://community.grafana.com/t/set---inputs-values-when-pushing-a-dashboard-json-config-to-grafana-via-the-api/2836)

It should have no effect on us

Reply:
suggest keeping these

Reply:
keep same structure with other repo

https://github.com/pingcap/tidb/blob/master/metrics/grafana/tidb.json

Reply:
fixed and i know why `__inputs` is gone


```
In Grafana v5.3.4+ the export modal has new checkbox for sharing for external use (other instances). If the checkbox is not checked then the __inputs section will not be included in the exported JSON file.
```